src/webview/create-component/createComponentLoader.ts

```diff
@@ -92,6 +92,9 @@ export default class CreateComponentLoader {
         );
         panel.webview.html = await loadWebviewHtml('createComponentViewer', panel);
         CreateComponentLoader.panel = panel;
+
+        panel.webview.postMessage({ action: 'initFromRootFolder', rootFolder: folderPath });
```

Comment:
It's possible that the webview hasn't loaded properly when this message is set, so it's unable to process this message, and the webview will be stuck on the loading package.

In practice, I sometimes get this the first time that I launch the webview.

This could be fixed by sending the message in the `'init'` message handler